### ISSUE TITLE
Questions about fine tuning

### ISSUE THREAD

**Issue by newn1u** [giddyyupp/VLMDiff#2]
My training process is as follows. Why is the loss so high

LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name            | Type                   | Params
-----------------------------------------------------------
0 | encoder         | Encoder                | 34.2 M
1 | decoder         | Decoder                | 49.5 M
2 | loss            | LPIPSWithDiscriminator | 17.5 M
3 | quant_conv      | Conv2d                 | 72    
4 | post_quant_conv | Conv2d                 | 20    
-----------------------------------------------------------
86.4 M    Trainable params
14.7 M    Non-trainable params
101 M     Total params
404.543   Total estimated model params size (MB)
Epoch 0:   0%|                                                    | 0/3628 [00:00<?, ?it/s]/home/syf/anaconda3/envs/vlmdiff/lib/python3.10/site-packages/pytorch_lightning/utilities/data.py:56: UserWarning: Trying to infer the `batch_size` from an ambiguous collection. The batch size we found is 1. To avoid any miscalculations, use `self.log(..., batch_size=batch_size)`.
  warning_cache.warn(
[ WARN:0@5.385] global loadsave.cpp:848 imwrite_ Unsupported depth image for selected encoder is fallbacked to CV_8U.
Epoch 0:  79%|██████████████████▏    | 2869/3628 [14:57<03:57,  3.20it/s, loss=1.93e+04, v_num=0, aeloss_step=6.96e+4, dEpoch 0:  79%|███▏| 2870/3628 [14:57<03:57,  3.20it/s, loss=1.93e+04, v_num=0, aeloss_step=6.96e+4, discloss_step=0.000]Epoch 0:  79%|███▏| 2870/3628 [14:57<03:57,  3.20it/s, loss=1.81e+04, v_num=0, aeloss_step=1.65e+4, discloss_stEpoch 0:  79%|▊| 2871/3628 [14:58<03:56,  3.20it/s, loss=1.81e+04, v_num=0, aeloss_step=1.65e+4, discloss_step=0.000]

**Comment by giddyyupp** [comment]
Hello, 
sorry for late response.

If you check the LPIPSWithDiscriminator loss implementation, it is a combination of losses such as Rec (L1), KL, Perceptual etc. Here the reconstruction loss makes the most of this large number as it is calculated on the pixel level i.e. calculating the diff on 256x256x3 numbers. So even the pixel-wise difference is very small it ends up with such big numbers. 
As long as the loss goes down during training it is fine. You can also check the reconstructed images during training to validate.
